### PR TITLE
Update policies.md

### PR DESCRIPTION
**Pull request by guce**
调整翻译语句

Signed-off-by: GU Ce <guce_baresi@163.com>

### CODE REVIEW

File: docs/locale/zh_CN/source/policies/policies.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ peer/ChaincodeToChaincode: /Channel/Application/Writers
 
 ## 在Fabric中如何写策略
 
-如果你想修改 Fabric 的任何东西，和资源相关的策略都描述了**谁**需要批准它，无论是明确使用某个还是一组身份的离线签名还是一组。在保险领域，一个明确的签名可以是业主保险代理集团中的一员。而一个隐含的签名类似于需要业主保险代理集团中的大多数管理成员批准。这很重要，因为集团中的成员可以在不更新策略的情况下变动。在 Hyperledger Fabric 中，策略中明确的签名使用 `Signature` 语法，隐含的签名使用 `ImplicitMeta` 语法。
+如果你想修改 Fabric 的任何东西，和资源相关的策略都描述了**谁**需要批准它，可以是个人的明确签名，也可以是组的隐含签名。在保险领域，一个明确的签名可以是业主保险代理集团中的一员。而一个隐含的签名类似于需要业主保险代理集团中的大多数管理成员批准。这很重要，因为集团中的成员可以在不更新策略的情况下变动。在 Hyperledger Fabric 中，策略中明确的签名使用 `Signature` 语法，隐含的签名使用 `ImplicitMeta` 语法。
```
Comment (davidkhala) [inline docs/locale/zh_CN/source/policies/policies.md]: 能帮忙把原文找出来方便大家审阅么？

Comment (guce) [reply docs/locale/zh_CN/source/policies/policies.md]: 好的

> If you want to change anything in Fabric, the policy associated with the resource describes who needs to approve it, either with an explicit sign off from individuals, or an implicit sign off by a group. In the insurance domain, an explicit sign off could be a single member of the homeowners insurance agents group.

Comment (davidkhala) [reply docs/locale/zh_CN/source/policies/policies.md]: 可以是来自个人的一个显式签名，也可以是组的一个隐式签名

Comment (guce) [reply docs/locale/zh_CN/source/policies/policies.md]: ok

**Review by davidkhala — APPROVED** [review]

**Pull request merged**